src/python/grpcio/grpc/_server.py

```diff
@@ -773,7 +776,11 @@ def cleanup_server(timeout):
                 _stop(state, timeout).wait()
 
         thread = _common.CleanupThread(
-            cleanup_server, target=_serve, args=(state,))
+            cleanup_server,
+            target=_serve,
+            args=(state,),
+            name='Thread-gRPC-CleanupServer',
```

Comment:
Ditto here (I didn't realize the cleanup was a secondary part), this should probably be called `Thread-gRPC-Serve` or something more specific about **what** is being served.

Reply:
If we're consistently naming threads after targets: `Thread-gRPC-_serve`?

This is the thread that drives gRPC Core on the service-side and does all the work of serving RPCs except for the running of application code (which is done in the thread pool supplied by the application).